identify/extensions.py

```diff
@@ -10,6 +10,7 @@
     'bash': {'text', 'shell', 'bash'},
     'bat': {'text', 'batch'},
     'bats': {'text', 'shell', 'bash', 'bats'},
+    'beam': {'binary', 'erlang'},
```

Comment:
I don't think this is quite right -- if one were to try and filter erlang source files and they got a `.beam` file I think the linter would be quite surprised?

Reply:
Ah sorry yeah you are right, I'll update.